README.md

```diff
@@ -2,62 +2,108 @@
+- Environment variables and credentials loaded from `.env` file or alternate path
 
-## Deploy the controller
-This command will start the API controller server into a local docker container.
-```shell
-make controller
-```
+## Getting Started
 
-## Initialize the database:
-The database is Initialized when building the controller.
-In order to erase and start fresh, we can simply use the controller's command line interface.
+### Pre-Requisites
+- Docker Desktop installed and updated
+- Host networking enabled (see [Docker documentation](https://docs.docker.com/network/drivers/host/#docker-desktop) for more information)
 
-```shell
-python -m controller.src.main initdb
-```
+### Deployment
+1. **Deploy the controller and UI**:
 
-## To start the application's API:
+    Run `make up` to start the API controller and UI in local Docker containers. If there are any port conflicts on your machine, update the ports in [docker-compose.yml](docker-compose.yml) and the URL's in [controller-config.yaml](controller/controller-config.yaml) as needed.
 
-```shell
-uvicorn pipeline:app
-```
+2. **Deploy an application**:
 
-## To start UI:
-Future work will include a UI command to run the UI.
-```shell
-make ui
-```
+    Follow the [quick start](examples/quick_start/notebook.ipynb) to get up and running quickly. It will involve the following steps:
+    1. Create a `.env` file which include OpenAI credentials
+    2. Configure parameters in [workflow-config.yaml](examples/quick_start/workflow-config.yaml) as needed including embedding model, completion model, generation parameters, port changes, and more.
+    3. Build your application in [workflow.py](examples/quick_start/workflow.py) using pre-defined components such as `SessionLoader`, `RefineQuery`, `MultiRetriever`, `HistorySaver`, `Agent`, and more.
+    3. Deploy the application via `genai-factory run --config-file workflow-config.yaml workflow.py`
+    4. Follow the [quick start](examples/quick_start/notebook.ipynb) for more details on how to ingest data and query the model via the UI.
 
-# CLI usage
+### Configuration
+There are three main types of configuration in the GenAI Factory:
 
-To ingest data into the vector database:
-```shell
-python -m controller.src.main ingest -l web https://milvus.io/docs/overview.md
-```
+1. **`.env`**:
 
-To ask a question:
-```shell   
-python -m controller.src.main infer "What is Milvus?"
-```
+    This is used to define credentials such as OpenAI API keys and define where to load [controller-config.yaml](controller/controller-config.yaml) and [workflow-config.yaml](examples/quick_start/workflow-config.yaml) files.
+
+    Example:
+    ```bash
+    OPENAI_BASE_URL=...
+    OPENAI_API_KEY=...
+    GENAI_FACTORY_IS_LOCAL_CONFIG=true  # Set to 'true' for local configuration
+    MLRUN_GENAI_LOCAL_CHROMA=/tmp/chromadb  # Path to local Chroma database (if applicable)
```

Comment:
I think the env vars prefixes were renamed from `MLRUN_GENAI` to `GENAI_FACTORY`